t/10-jobs-referal.t

```diff
@@ -31,6 +31,9 @@ sub job_is_linked ($job) {
     $job->comments->find({text => {like => 'label:linked%'}}) ? 1 : 0;
 }
 
+$schema->txn_begin;
+my $scheduled_products_rs = $schema->resultset('ScheduledProducts')->create({id => 30, settings => []});
+
```

Comment:
unrelated

Reply:
should the beginning of the subtest be after a blankline for visibility or..?